redis/cache/redisx/request.go

```diff
@@ -249,18 +250,19 @@ func (r Request) setStructValue(dst reflect.Value, src reflect.Value) error {
 func (r Request) convertAndSetByteSlice(dst reflect.Value, src reflect.Value) error {
 	asBytes, _ := src.Interface().([]byte)
 	asStr := string(asBytes)
-	switch dst.Kind() {
-	case reflect.Int64:
```

Comment:
Why did you change this switch?

Reply:
@rbietin Because in my case I have to check `dst.Type()` and not `dst.Kind()` that is checked in other cases.

Reply:
@fishy @kylelemons Ok, changed back to using `switch`.